tools/docker-compose/README.md

```diff
@@ -1,8 +1,50 @@
-docker build --no-cache=true --rm=true -t ansible/awx_devel:latest .
-docker run --name awx_test -it --memory="4g" --cpuset="0,1" -v /Users/meyers/ansible/:/awx_devel -p 8013:8013 -p 8080:8080 -p 27017:27017 -p 2222:22 ansible/awx_devel
+# Docker Compose for Dev container
+
+## How to start the Dev container
+
+In the root directory of your awx clone, run the following to build your docker image:
+
+```
+make docker-compose-build
+```
+
+Copy over your local settings 
+
+```
+cp awx/settings/local_settings.py.docker_compose awx/settings/local_settings.py
+```
+
+Build the UI
+
+```
+make ui-devel
+```
+
+Run the container 
+
+```
+make docker-compose
+```
+
+The app should now be accessible in your browser at `https://localhost:8043/#/home`
+
 
 ## How to use the logstash container
 
+#### Modify the docker-compose.yml
+
+Uncomment the following lines in the `docker-compose.yml`
+
+```
+#- logstash
+...
+
+#logstash:
+#  build:
+#    context: ./docker-compose
+#    dockerfile: Dockerfile-logstash
```

Comment:
this logstash thing was kind of a different animal than the full ELK stack tooling lying around.

If you're testing that logs can be consumed, you don't need the other components, and testing that it's in the text file from logstash is enough. We had that container running by default at some point, but this is still something that makes sense.

Reply:
Ok, great!  I tested out rsyslog with just the logstash container and logs are getting written to `/logstash.log` correctly.